addons/core/functions/fnc_getLrSettings.sqf

```diff
@@ -86,6 +86,18 @@ if (isNil "_radioCode") then {
     _value set [TFAR_CODE_OFFSET, _radioCode];
 };
 
+private _halfDuplexOverride = _value select HALFDUPLEX_OVERRIDE_OFFSET;
+if (isNil "_halfDuplexOverride") then {
+    _halfDuplexOverride = false; //Override is disabled by default. No harm done if we don't find the config value since it's expected default behaviour anyways
+    private _override = [_radio_object, "tf_halfDuplexOverride"] call TFAR_fnc_getLrRadioProperty;
```

Comment:
doesn't `getLrRadioProperty` also have a parameter for default value that you can use?